hyperon_das_atomdb/adapters/server_db.py

```diff
@@ -0,0 +1,277 @@
+import json
+from typing import Any, Dict, List, Optional, Tuple
+
+from requests import Session, exceptions
+
+from hyperon_das_atomdb.exceptions import (
+    ConnectionServerException,
+    NodeDoesNotExistException,
+)
+from hyperon_das_atomdb.i_database import IAtomDB
+from hyperon_das_atomdb.utils.decorators import retry
+
+
+class ServerDB(IAtomDB):
+    """A concrete implementation using servers databases.
+    AwsLambda and OpenFaas"""
+
+    def __repr__(self) -> str:
+        return "<Atom database Server>"  # pragma no cover
+
+    def __init__(
+        self,
+        database_name: str = 'das',
+        ip_address: str = None,
+        port: Optional[str] = None,
+    ) -> None:
+        if not ip_address:
+            raise ConnectionServerException(
+                message='You must send ip_address parameter'
+            )
+        self.database_name = database_name
+        self.ip = ip_address
```

Comment:
`self.ip` is not a good name. We may need to use domain names, as we have already discussed. perhaps `self.host` would be a better choice.

Anyway, it should have a default just like `self.port`. Perhaps `localhost` or `0.0.0.0`. I prefer the former.

Reply:
Well, in our first discussion we agreed that users should send IP and PORT as IP is required. But if we use domain in future I will change it because self.ip is not so good in this context.

I don't understand your second comment. I put port 8080 in case users don't send it because Levi told us that it is a standard port in OpenFaas and in AWS this information is not necessary.

Reply:
Exactly because we can choose to use IP or domain, `host` is a better name since it applies to both. As for my second comment, I was arguing that you should have a default for `ip` (or `host`) as well.